src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -313,19 +304,16 @@ public void put(final Collection<SinkRecord> records) {
   public Map<TopicPartition, OffsetAndMetadata> preCommit(
       Map<TopicPartition, OffsetAndMetadata> offsets) throws RetriableException {
     long startTime = System.currentTimeMillis();
-    LOGGER.debug(
-        Logging.logMessage("SnowflakeSinkTask[ID:{}]:preCommit {}", this.id, offsets.size()));
+    LOGGER.debug("SnowflakeSinkTask[ID:{}]:preCommit {}", this.id, offsets.size());
 
     // return an empty map means that offset commitment is not desired
     if (sink == null || sink.isClosed()) {
       LOGGER.warn(
-          Logging.logMessage(
-              "SnowflakeSinkTask[ID:{}]: sink " + "not initialized or closed before preCommit",
-              this.id));
+          "SnowflakeSinkTask[ID:{}]: sink " + "not initialized or closed before preCommit",
```

Comment:
nit: extra " + " not needed in between. (Probably some gfmt thing) 
Lets fix this.